extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/conf/ConfigurationService.java

```diff
@@ -85,4 +85,40 @@ public PrivateKey getClearpassKey() throws GuacamoleException {
         return environment.getProperty(CASGuacamoleProperties.CAS_CLEARPASS_KEY);
     }
 
+    /**
+     * Returns the attribute used to determine group memberships
+     * in CAS, or null if not defined.
+     *
+     * @return
+     *     The attribute name used to determine group memberships in CAS,
+     *     null if not defined.
+     *
+     * @throws GuacamoleException
+     *     If guacamole.properties cannot be parsed.
+     */
+    public String getGroupAttribute() throws GuacamoleException {
+        return environment.getProperty(CASGuacamoleProperties.CAS_GROUP_ATTRIBUTE);
+    }
+
+    /**
+     * Returns the attribute used to format group DN's
+     * in CAS, or null if not defined.  If CAS is backed by LDAP, it will
+     * return an LDAP DN, such as CN=foo,OU=bar,DC=example,DC=com.
+     * This attribute may be set to CN=%s,OU=bar,DC=example,DC=com and given
+     * the example above, would result in a group called "foo."  CAS backed
+     * by something other than LDAP (such as a database) would likely not 
+     * need this.
```

Comment:
As above, I'm not sure this is correct. How is the format itself an attribute?

Reply:
I'd also like to point out that the line wrapping is a bit wonky here. We normally wrap our comments at 80 columns, but here the pattern of wrapping is inconsistent.

Reply:
Clarified the comment and fixed line wrapping